db/compaction_picker.cc

```diff
@@ -156,9 +91,9 @@ void CompactionPicker::ReleaseCompactionFiles(Compaction* c, Status status) {
   }
 }
 
-void CompactionPicker::GetRange(const CompactionInputFiles& inputs,
```

Comment:
Why not keep the overloaded GetRange name for all three functions? I am curious about your thought process here. 
I do understand that your two overloaded GetCombinedRangeFromInputs are internally using GetRangeFromInputs, but then again, they are all just getting ranges in the end. 
(Either way, it is fine with me).